app/invocation/invocation_action_card.tsx

```diff
@@ -0,0 +1,70 @@
+import pako from "pako";
+import React from "react";
+import format from "../format/format";
+import InvocationModel from "./invocation_model";
+import { build } from "../../proto/remote_execution_ts_proto";
+import rpcService from "../service/rpc_service";
+
+interface Props {
+  model: InvocationModel;
+  inProgress: boolean;
+  search: string;
+}
+
+interface State {
+  loading: boolean;
+  contents: ArrayBuffer;
+  action_array: Uint8Array;
+  action_obj: build.bazel.remote.execution.v2.Action;
```

Comment:
Contents and action_array don't need to be stored in state, only action_obj does

nit: rename `action_obj` to `action`